controllers/BaseIndexController.php

```diff
@@ -32,9 +32,8 @@ public function __construct(
     public function model(): Listing
     {
         return $this->model
-        ->withCount(5)
```

Comment:
There is no need to set the number of values in controller so this method is deleted.